### PR TITLE
files: add create_buckets cli

### PR DESCRIPTION
**Pull request by harunurhan**

### CODE REVIEW

File: backend/inspirehep/files/api/s3.py [hunk 1]
```diff
@@ -18,14 +18,28 @@ def __init__(self, client, resource):
...
 
         :param key: the file key
         :return: bucket: The corresponding bucket.
         """
-        bucket = f"{current_app.config.get('S3_BUCKET_PREFIX')}{key[0]}"
-        return bucket
+        return S3.get_prefixed_bucket(key[0])
+
+    @staticmethod
+    def get_prefixed_bucket(bucket_without_prefix):
```
Comment (michamos) [inline backend/inspirehep/files/api/s3.py]:
```suggestion
    def get_prefixed_bucket_name(bucket_without_prefix):
```

Comment (harunurhan) [reply backend/inspirehep/files/api/s3.py]: well bucket name, called `bucket` in other parts, that's why I kept it like that.

File: backend/inspirehep/files/cli.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+@files.command(help="Creates S3 buckets for [0-9] and [a-f]")
+@with_appcontext
+def create_buckets():
+
+    click.secho("Creating buckets")
+
+    for bucket in BUCKETS:
+        current_s3_instance.create_bucket(bucket)
```
Comment (drjova) [inline backend/inspirehep/files/cli.py]: I'd print also which bucket created

File: backend/inspirehep/files/cli.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        click.secho(f"Created bucket: ${bucket}")
```
Comment (drjova) [inline backend/inspirehep/files/cli.py]: this is not JS :P

```suggestion
        click.secho(f"Created bucket: {bucket}")
```

**Pull request merged**